client/client_test.go

```diff
@@ -193,81 +191,116 @@ func TestHttpClientUrl(t *testing.T) {
 }
 
 // Test that our loaded certificates include the system CAs, and our own.
-func TestCaLoading(t *testing.T) {
-	conf := Config{
-		ServerCert: "server.crt",
-	}
-
-	certs := loadServerTrust(&conf)
-
-	// Verify that at least one of the certificates belong to us, and one
-	// belongs to a well known certificate authority.
-	var systemOK, oursOK bool
-	subj := certs.Subjects()
-	for i := 0; i < len(subj); i++ {
-		if strings.Contains(string(subj[i]), "VeriSign, Inc.") {
-			systemOK = true
-		}
-		// "Acme Co", just a dummy certificate in this repo.
-		if strings.Contains(string(subj[i]), "Acme Co") {
-			oursOK = true
+func TestCALoading(t *testing.T) {
```

Comment:
shall we consider adding a test for the chain of certificates? (to test `loadClientTrust` logic?)

Reply:
I think that is a good idea actually, yeh. Let's!